frontend/app/components/encryptable/form.js

```diff
@@ -16,10 +17,51 @@ export default class Form extends BaseFormComponent {
   @tracked selectedTeam;
   @tracked assignableTeams;
 
-  @tracked hasErrors;
+  @tracked Errors;
 
   AccountValidations = AccountValidations;
 
+  //set display field to true when attribute is set, for creating username is default
+  @tracked
+  isUsernameFieldActive = this.record.cleartextUsername || this.isNewRecord;
+
+  //set display field to true when attribute is set, for creating password is default
+  @tracked
+  isPasswordFieldActive = this.record.cleartextPassword || this.isNewRecord;
+
+  //set display field to true when attribute is set
+  @tracked
+  isPinFieldActive = this.record.cleartextPin;
+
+  //set display field to true when attribute is set
+  @tracked
+  isTokenFieldActive = this.record.cleartextToken;
+
+  //set display field to true when attribute is set
+  @tracked
+  isEmailFieldActive = this.record.cleartextEmail;
+
+  //set display field to true when attribute is set
+  @tracked
+  isCustomAttrFieldActive = this.record.cleartextCustomAttr;
+
+  //create array proxy that ember checks the changes in each loop for dropdown, ember's rendering engine
+  // doesn't get notified by using a normal array
+  @tracked
+  items = A(
```

Comment:
A

(is it really just `A`? ... seems way too funny to me :joy: )

Reply:
Seems like it is...

https://api.emberjs.com/ember/release/classes/ArrayProxy

Reply:
> A
> 
> (is it really just `A`? ... seems way too funny to me joy )

it's JavaScript :joy: